testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksPartitioningCompatibility.java

```diff
@@ -79,7 +79,8 @@ private void testDatabricksCanReadFromCtasTableCreatedByTrinoWithSpecialCharacte
         }
     }
 
-    @Test(groups = {DELTA_LAKE_DATABRICKS, PROFILE_SPECIFIC_TESTS})
+    // Disabled due to flakiness per https://nineinchnick.github.io/trino-cicd/reports/flaky/
+    @Test(groups = {DELTA_LAKE_DATABRICKS, PROFILE_SPECIFIC_TESTS}, enabled = false)
```

Comment:
I don't think disabling this test helps to fix Retry-After failure. It happens in several places.